README.md

```diff
@@ -1,26 +1,63 @@
+# Quaver 
+
+[![Build Status](https://travis-ci.com/Quaver/Quaver.svg?branch=develop)](https://travis-ci.com/Quaver/Quaver) [![CodeFactor](https://www.codefactor.io/repository/github/swan/quaver/badge)](https://www.codefactor.io/repository/github/swan/quaver) [![license](https://img.shields.io/badge/license-Mozilla%20Public%20License%202.0-blue)](https://github.com/Quaver/Quaver/blob/develop/LICENSE) [![Discord](https://discordapp.com/api/guilds/354206121386573824/widget.png?style=shield)](https://discord.gg/nJa8VFr) <a href="https://twitter.com/intent/follow?screen_name=QuaverGame">
+        <img src="https://img.shields.io/twitter/follow/QuaverGame?style=social&logo=twitter"
+            alt="follow on Twitter"></a> [![stargazers](https://img.shields.io/github/stars/Quaver/Quaver?style=social)](#)
+
 <p align="center"> 
-  <img src="https://i.imgur.com/UK3PWJW.png" width="175px" height="175px">
+  <img src="https://i.imgur.com/AohWq5l.png">
 </p>
 
-# Quaver [![Build Status](https://travis-ci.com/Quaver/Quaver.svg?branch=develop)](https://travis-ci.com/Quaver/Quaver) [![CodeFactor](https://www.codefactor.io/repository/github/swan/quaver/badge)](https://www.codefactor.io/repository/github/swan/quaver) [![Blog](https://img.shields.io/badge/Blog-Read-blue.svg)](https://blog.quavergame.com) [![Discord](https://discordapp.com/api/guilds/354206121386573824/widget.png?style=shield)](https://discord.gg/nJa8VFr)
+| [![steam](https://i.imgur.com/rR4p9mW.png)](https://store.steampowered.com/app/980610/Quaver/) | [![twitter](https://i.imgur.com/Kd0SXlh.png)](https://twitter.com/QuaverGame) | [![discord](https://i.imgur.com/YrBmbEf.png)](https://discord.gg/nJa8VFr) | [![blog](https://i.imgur.com/o0fq6MA.png)](https://blog.quavergame.com) | [![website](https://i.imgur.com/svZCnuI.png)](https://quavergame.com) | 
+| --- | --- | --- | --- | --- |
+| **Steam Store Page** | **Follow Us On Twitter** | **Join The Discord** | **Read The Developer Blog** | **Visit The Website** |
```

Comment:
![image](https://user-images.githubusercontent.com/1794388/62318468-c4951a00-b450-11e9-91f4-89e4ed75aef9.png)
Could you change the text so it all fits on one line?

Reply:
Also pls use higher res images, or even better svgs, they look blurry on my 2x scale screen